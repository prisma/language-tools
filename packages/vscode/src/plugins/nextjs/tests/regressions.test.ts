import { TestContext } from './__helpers__/context'
import { regression_602 } from './__fixtures__/regressions'
import { runTestProject } from './__helpers__/testProject'

const ctx = new TestContext()

describe('regressions - ', () => {
  beforeEach(async () => {
    await ctx.beforeEach()
  })

  afterEach(async () => {
    await ctx.afterEach()
  })
  test('regression', async () => {
    const testDir = await ctx.setup({
      fs: { 'test.ts': regression_602 },
    })

    await runTestProject(testDir, 'test.ts')
    await ctx.snapshotFiles()
  })
})
