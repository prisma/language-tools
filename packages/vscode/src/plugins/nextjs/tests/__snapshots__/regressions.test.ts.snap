// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`regressions -  regression 1`] = `
import { NextApiRequest, NextApiResponse } from 'next'

import { primsa } from "~/lib/db";
import { requestWrapper } from "~/lib/auth/jwt";

export default async (req: NextApiRequest, res) => {
await requestWrapper(req, res, async (token: any) => {
  const posts = await prisma.post.findMany();
  res.status(200).json()({posts})
})
}
`;
